### DISCUSSION TITLE
Separate "Save Bookmark" Button in Context Menu for Improved User Experience

### DISCUSSION THREAD

**Discussion by JijinJayakumar** [dailydotdev/daily#911] [Feature Requests]
I would like to request the addition of a separate and easily identifiable "Save to Bookmark" button in the context menu in  application. This feature is essential for users who prefer to navigate quickly and may unintentionally click on undesired options.

Often, when users open new tabs and come across interesting topics they want to read later, they hastily attempt to add them to bookmarks. However, due to the proximity of other menu options, such as the "Do Not Show Post From..." link, they may accidentally click on the wrong option. This can lead to frustration and loss of valuable content.

By introducing a distinct button for saving items to bookmarks, we can greatly enhance the user experience and prevent accidental clicks on undesired options. This dedicated button will allow users to efficiently bookmark intriguing topics, articles, or posts for later reference, regardless of their speed or navigation habits.

**Comment by idoshamun** [comment]
Hi @JijinJayakumar,

By a dedicated button you mean a new button on the card itself?

**Reply by JijinJayakumar** [reply]
@idoshamun  It would be great if you could include a dedicated bookmark button within the card itself, or alternatively, make it the first option in the dropdown menu. Having this convenient and helpful feature would be a valuable addition.

**Reply by nimrodkra** [reply]
@JijinJayakumar done :)

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/f219841d-0c9b-445b-a618-522b776c07d2)
